examples/arm/run.sh

```diff
@@ -5,7 +5,7 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-set -eu
+set -eux
```

Comment:
add `-x` only for CI, this will make it too verbose for end user running this manually